ppafm/io.py

```diff
@@ -374,15 +389,19 @@ def loadGeometry(fname=None,params=None):
             lvec[ 2,:  ] = params['gridB'].copy()
             lvec[ 3,:  ] = params['gridC'].copy()
         atoms = [list(Zs), list(xyzs[:, 0]), list(xyzs[:, 1]), list(xyzs[:, 2]), list(qs)]
-    elif(is_cube):
+    elif(format=="cube"):
         atoms = loadAtomsCUBE(fname)
         lvec  = loadCellCUBE(fname)
         nDim  = loadNCUBE(fname)
-    elif(is_xsf):
-        atoms, nDim, lvec = loadXSFGeom( fname)
-    elif(is_npy):
-        atoms, nDim, lvec = loadNPYGeom( fname) # under development
-        #TODO: introduce a function which reads the geometry from the .npy file
+    elif(format=="xsf"):
+        atoms, nDim, lvec = loadXSFGeom(fname)
+    elif(format=="npy"):
+        atoms, nDim, lvec = loadNPYGeom(fname) # under development
+    elif(format=="poscar"):
+        atoms, nDim, lvec = loadPOSCAR(fname)
+    elif(format=="in"):
+        atoms, nDim, lvec = loadGeometryIN(fname)
+    #TODO: introduce a function which reads the geometry from the .npy file
     else:
         sys.exit("ERROR!!! Unknown format of geometry system. Supported formats are: .xyz, .cube, .xsf \n\n")
```

Comment:
The message is wrong (we support more formats), and it would be better if this was a `raise ValueError()` instead of `sys.exit()`, so that if you use this function in some place that can catch the error, it won't just suddenly exit the whole program.